items-service/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+server.port=9091
```

Comment:
can this port be dynamic.

Reply:
https://www.javacodemonk.com/setting-a-random-port-in-spring-boot-application-at-startup-87022e01

here:
https://www.onlinetutorialspoint.com/spring-boot/how-to-use-spring-boot-random-port.html